CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Okta Python SDK Changelog
 
+## v2.0.0
+- Regenerate code using the [open API spec v2.5.0](https://github.com/okta/okta-management-openapi-spec/releases/tag/openapi-2.5.0).
+- Make sign_on_mode of all apps instance of enum, issue #198 (this change might be not compatible with custom clients code, although probability is low)
+- Add response headers to OktaAPIResponse object, issue #218
+
+_New resources:_
+* Domain
+* UserSchema
+
+_New models:_
+* DnsRecord
+* DnsRecordType
+* Domain
+* DomainCertificate
+* DomainCertificateMetadata
+* DomainCertificateSourceType
+* DomainCertificateType
+* DomainListResponse
+* DomainValidationStatus
+* UserSchemaAttributeEnum
+* UserSchemaAttributeItems
+* UserSchemaAttributeMasterPriority
+* UserSchemaAttributeMasterType
+* UserSchemaAttributeScope
+* UserSchemaAttributeType
+* UserSchemaAttributeUnion
+* UserSchemaProperties
+* UserSchemaPropertiesProfile
+* UserSchemaPropertiesProfileItem
+
+### Breaking changes
+Previously, some predefined apps set string type for attribute `sign_on_mode`, while some other apps, including generic app, leave ApplicationSignOnMode type.
+To make type consistent, sign_on_mode attribute of all applications are ApplicationSignOnMode type now. Thus, code like the following, which was working previously, won't provide desired result:
```

Comment:
We have now standardized the `sign_on_mode` attribute making its type an `ApplicationSignOnMode`. Thus, code like the following, which was working previously, won't provide desired result: